src/config.ts

```diff
@@ -0,0 +1,131 @@
+import * as autoproj from './autoproj';
+import * as context from './context';
+import * as path from 'path';
+import * as fs from 'fs'
+
+export const C_CPP_PROPERTIES_JSON = `{
+    "configurations": [
```

Comment:
From https://github.com/Microsoft/vscode-cpptools/blob/master/Extension/c_cpp_properties.schema.json, it seems that the only thing you need to set is compileCommands and version.

Reply:
You are probably right. Yet, cpptools itself creates it like this (I just copied their default here). But I will remove everything to keep it cleaner 👍

Reply:
Updated.